scripts/src/java/org/oppia/android/scripts/label/AccessibilityLabelCheck.kt

```diff
@@ -5,126 +5,145 @@ import org.w3c.dom.Node
 import org.w3c.dom.NodeList
 import java.io.File
 import java.io.FileInputStream
-import java.util.stream.Collectors
-import java.util.stream.IntStream
 import javax.xml.parsers.DocumentBuilderFactory
 
 /**
  * Script for ensuring that all the Activities in the codebase are defined with accessibility
  * labels.
  *
  * Usage:
- *   bazel run //scripts:accessibility_label_check -- <path_to_directory_root> <path_to_app_level
- *   _manifest_file>
+ *   bazel run //scripts:accessibility_label_check -- <path_to_directory_root> <path_to_manifest
+ *   _files>
  *
  * Arguments:
  * - path_to_directory_root: directory path to the root of the Oppia Android repository.
- * - path_to_app_level_manifest_file: path to the manifest file of app layer.
+ * - path_to_manifest_files: path to the manifest files
  *
  * Example:
  *   bazel run //scripts:accessibility_label_check -- $(pwd) app/src/main/AndroidManifest.xml
  */
 fun main(vararg args: String) {
   val repoPath = "${args[0]}/"
 
-  val accessibilityLabelExemptiontextProto = "scripts/assets/accessibility_label_exemptions"
+  val accessibilityLabelExemptionTextProto = "scripts/assets/accessibility_label_exemptions"
 
   val accessibilityLabelExemptionList = loadAccessibilityLabelExemptionsProto(
-    accessibilityLabelExemptiontextProto
+    accessibilityLabelExemptionTextProto
   ).getExemptedActivityList()
 
-  val manifesFilePath = args[1]
+  val manifestPaths = args.drop(1)
 
-  val fullPathToManifestFile = repoPath + manifesFilePath
+  val activityPathPrefix = "app/src/main/java/"
 
   val builderFactory = DocumentBuilderFactory.newInstance()
 
-  val docBuilder = builderFactory.newDocumentBuilder()
-
-  val doc = docBuilder.parse(File(fullPathToManifestFile))
-  doc.getDocumentElement().normalize()
-
-  val completeActivityList = convertNodeListToListOfNode(doc.getElementsByTagName("activity"))
-
-  val activityListWithoutLabel = completeActivityList.filter { node ->
-    checkIfActivityHasMissingLabel(node, accessibilityLabelExemptionList)
+  val missingAccessibilityLabelActivities = manifestPaths.flatMap { relativePath ->
+    val file = File(repoPath + relativePath)
+    val docBuilder = builderFactory.newDocumentBuilder()
+    val doc = docBuilder.parse(file)
+    doc.getDocumentElement().normalize()
+    val packageName = doc.getDocumentElement().getAttribute("package")
+    doc.getElementsByTagName("activity").toListOfNodes().mapNotNull { activityNode ->
```

Comment:
```suggestion
    return@flatMap doc.getElementsByTagName("activity").toListOfNodes().mapNotNull { activityNode ->
```

Optional, but I find the extra ``return`` can help with clarity for longer lambdas.

Reply:
Added.